### PR TITLE
Merge master branch 'origin/master' into scheduler-manager

### PR DESCRIPTION
**Pull request by TGrigor**

### CODE REVIEW

**Review by anush1111 — APPROVED** [review]

**Review by Tavrizyan — APPROVED** [review]

File: src/SchedulerManager/Travely.SchedulerManager.API/Properties/launchSettings.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "dotnetRunMessages": "true",
-      "applicationUrl": "http://localhost:5300;https://localhost:5301"
```
Comment (GevorgMartirosyan) [inline src/SchedulerManager/Travely.SchedulerManager.API/Properties/launchSettings.json]: please do not change the ports.

File: src/SchedulerManager/Travely.SchedulerManager.API/Startup.cs [hunk 2]
```diff
@@ -4,43 +4,42 @@
...
             services.AddCors(options =>
             {
-                options.AddDefaultPolicy(
+                options.AddPolicy("CORS",
                     builder =>
                     {
                         builder.AllowAnyHeader()
                             .AllowAnyMethod()
                             .AllowCredentials()
-                            .WithOrigins("http://localhost:3000", "https://travely.am");
```
Comment (GevorgMartirosyan) [inline src/SchedulerManager/Travely.SchedulerManager.API/Startup.cs]: Please do not change Infrastructure related statements.

File: src/SchedulerManager/Travely.SchedulerManager.API/Startup.cs [hunk 3]
```diff
@@ -50,31 +49,33 @@ public void ConfigureServices(IServiceCollection services)
...
             app.UseJobClient();
             app.UseNotifier();
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapGrpcService<ReminderService>();
+                endpoints.MapGrpcService<EmailService>();
                 endpoints.MapGet("/", async context => await context.Response.WriteAsync("Service running"));
             });
+
+            //xx.StartJobAsync(new InformationJob(), new InformationJobParameter { TourName = "Enqueue Job" }).GetAwaiter().GetResult();
```
Comment (GevorgMartirosyan) [inline src/SchedulerManager/Travely.SchedulerManager.API/Startup.cs]: Please remove this.

File: src/SchedulerManager/Travely.SchedulerManager.API/Startup.cs [hunk 4]
```diff
@@ -50,31 +49,33 @@ public void ConfigureServices(IServiceCollection services)
...
+            //yy.StartJobAsync(new InformationJob(), TimeSpan.FromSeconds(5), new InformationJobParameter { TourName = "Scheduled Job" }).GetAwaiter().GetResult();
```
Comment (GevorgMartirosyan) [inline src/SchedulerManager/Travely.SchedulerManager.API/Startup.cs]: This also.

File: src/SchedulerManager/Travely.SchedulerManager.API/appsettings.json [hunk 5]
```diff
@@ -14,23 +14,29 @@
...
     }
   },
-
+  "SeriLog": {
+    "LogTable": "logs",
+    "ConnectionString": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=SchedulerDb;Integrated Security=True"
+  },
   "RepositoryOptions": {
-    "ConnectionString": "Server=sql-server-db;Database=SchedulerDb;User=sa;Password=Passw0rd!;"
+    "ConnectionString": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=SchedulerDb;Integrated Security=True"
```
Comment (GevorgMartirosyan) [inline src/SchedulerManager/Travely.SchedulerManager.API/appsettings.json]: Please do not change this.

File: src/SchedulerManager/Travely.SchedulerManager.API/appsettings.json [hunk 6]
```diff
@@ -14,23 +14,29 @@
...
   },
 
   "JobOptions": {
-    "ConnectionString": "Server=sql-server-db;Database=SchedulerDb;User=sa;Password=Passw0rd!;"
+    "ConnectionString": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=SchedulerDb;Integrated Security=True"
```
Comment (GevorgMartirosyan) [inline src/SchedulerManager/Travely.SchedulerManager.API/appsettings.json]: Please do not change this.

**Review by GevorgMartirosyan — CHANGES_REQUESTED** [review]

**Review by GevorgMartirosyan — APPROVED** [review]

**Pull request merged**